WcaOnRails/app/webpacker/components/StaticPages/About.js

```diff
@@ -51,7 +53,11 @@ function About({ currentBoardMembers }) {
           <I18nHTMLTranslate i18nKey="about.who_we_are_content_html" />
         </div>
         <div className="col-md-4">
-          <img className="about-page-image" src="/assets/about/bulgarian-open.jpg" alt="" />
```

Comment:
If that image is going to be used only in the react parts of the website, wouldn't it work to move them to `app/webpacker/images` (or even in the local component folder actually) and import them directly in `About.js`?
I think `import bulgarianOpen from './bulgarian-open.jpg'` should work if it's in the same folder, but maybe I'm missing something.